src/components/serviceRequest/ClientInformationSection.tsx

```diff
@@ -74,6 +108,8 @@ export default function ClientInformationSection(props: ClientInformationSection
                 label={clientInformationLabels.FirstName}
                 placeholder={clientInformationLabels.FirstName}
                 onChange={(e) => setFirstName(e.target.value)}
+                onBlur={() => validate('first_name')}
+                invalid={errors.first_name}
```

Comment:
I'm seeing an error on this line - `**Property 'first_name' does not exist on type '{}'.**`